doc/src/conf.py

```diff
@@ -31,7 +31,10 @@
     'sphinx.ext.graphviz',
     'sphinx.ext.inheritance_diagram',
     'sphinx.ext.mathjax',
```

Comment:
Is this needed?

Reply:
I do use Mathjax in one place, I don't think there's any harm in keeping it.